docs/Actions/send-email.md

```diff
@@ -0,0 +1,531 @@
+| DNN email template to use.    | Choose from a predefined email from the system                                                                                                                                                                                                                            | No              | Unset                                    |
+| Body                          | The email template you want to send                                                                                                                                                                                                                                       | Yes             | `empty string`                           |
+| Headers                       | Append custom headers to the email.                                                                                                                                                                                                                                       | Yes             | Unset                                    |
+| Attach Portal Files           | Pick a file that you want to send as attachment via email                                                                                                                                                                                                                 | No              | Unset                                    |
+| Attach Field #                | Form only parameters, provide the ability to add up to 5 Single File Uploads fields to be send as attachments to the email                                                                                                                                                | No              | Unset                                    |
+| Attach from Token             | This lets you compute file path from token. It must be relative to portal home folder. Multiple items must be separated by semicolons.                                                                                                                                    | Yes             | `empty string`                           |
+| Ignore Errors                 | If errors are ignored, the submission of the form continues even when there is an error sending the email. Otherwise, an error message will be displayed to the user. Note that if the email is being sent async or is placed in a pickup folder, errors won't be caught. | No              | No                                       |
+
+## `Security`
+**Note that this presents a security risk when the To, From, CC, BCC and Reply To parameter has content from untrusted sources.**

```

Comment:
Maybe use :::important or :::caution elements here instead of bold?
Also, mentioning tokens instead of just content would add more clarity?

Reply:
nice...